utils/utils.go

```diff
@@ -120,6 +120,10 @@ func FindCommand(def string, options []string) string {
 			return path
 		}
 	}
+	// Additional check for K0s binary
+	if def == "k0s" {
```

Comment:
Looking at [how it's used here](https://github.com/kairos-io/kairos-sdk/blob/2bd0cc2843b8db1ec566924344a576579ad1fd2a/system/grub.go#L38), `def` is meant to be the default command if none of the commands in `options` is found in `PATH`. I'm not sure why we expect the default to work if it's not in `PATH` but in any case, you don't seem to need `K0sBin` function. The new code is equivalent to this:

```
FindCommand("",  []string{"k0s"})
```

meaning: "go find k0s command in `PATH` and if none if found, return empty string", which is what the code here is doing as long as `/usr/bin` and `/usr/local/bin` are in `PATH`